generators/index.js

```diff
@@ -230,10 +230,10 @@ module.exports = generators.Base.extend({
         version: '0.0.1',
         main: 'server/server.js',
         scripts: {
-          'package': 'npm install; npm run client:build',
+          'package': 'yarn',
```

Comment:
As I said, the `client:build` task is unnecessary here since there is a `postinstall` task doing it.